StationeersLaunchPad/LaunchPadConfig.cs

```diff
@@ -39,18 +42,29 @@ public static class LaunchPadConfig
     public static List<ModInfo> Mods = new();
     public static HashSet<string> GameAssemblies = new();
     public static LoadState LoadState = LoadState.Initializing;
+    public static bool AutoUpdate = false;
     public static bool AutoLoad = true;
+    public static bool HasUpdated = false;
     public static Stopwatch AutoStopwatch = new();
 
     public static async void Run()
     {
+      AutoUpdate = AutoUpdateOnStart.Value;
       AutoLoad = AutoLoadOnStart.Value;
-
+      
       await Load();
 
-      while (LoadState < LoadState.Configuring)
+      while (LoadState < LoadState.Updating)
         await UniTask.Yield();
 
+      if (HasUpdated)
+      {
+        Mods = new();
+        LoadState = LoadState.ModsLoaded;
+        AutoLoad = false;
```

Comment:
If we are on dedicated server, this will just hang forever, since there is no way to trigger a load manually (this is definitely a problem everywhere else we stop autoload as well). do you think on dedicated server we should just exit immediately here?